### PR TITLE
SAW - Direct Cost Made Configurable

### PR DESCRIPTION
**Pull request by SamWord**
Total Direct Costs are only displayed when USE_INDIRECT_COST is true, Total Indirect Costs are no longer displayed at all.

### CODE REVIEW

File: app/helpers/service_calendar_helper.rb [hunk 1]
```diff
@@ -87,11 +81,6 @@ def display_total_direct_cost_per_study_pppvs service_request
     currency_converter sum
   end
 
-  def display_total_indirect_cost_per_study_pppvs service_request
```
Comment (jayhardee9) [inline app/helpers/service_calendar_helper.rb]: Isn't this being used at `app/views/service_requests/calendar_totals.html.haml:44`?

File: app/helpers/service_calendar_helper.rb [hunk 2]
```diff
@@ -116,11 +105,6 @@ def display_total_direct_cost_per_study_otfs service_request, line_items
     currency_converter sum
   end
 
-  def display_total_indirect_cost_per_study_otfs service_request, line_items
```
Comment (jayhardee9) [inline app/helpers/service_calendar_helper.rb]: Ditto. app/views/service_requests/calendar_totals.html.haml:62

File: app/helpers/service_calendar_helper.rb [hunk 3]
```diff
@@ -139,11 +123,6 @@ def display_grand_total_direct_costs service_request, line_items
     currency_converter sum
   end
 
-  def display_grand_total_indirect_costs service_request, line_items
```
Comment (jayhardee9) [inline app/helpers/service_calendar_helper.rb]: app/views/service_requests/calendar_totals.html.haml:81
app/views/service_requests/review/_review_totals.html.haml:32

File: app/helpers/service_calendar_helper.rb [hunk 4]
```diff
@@ -154,11 +133,6 @@ def display_study_grand_total_direct_costs protocol, service_request
     currency_converter sum
   end
 
-  def display_study_grand_total_indirect_costs protocol, service_request
```
Comment (jayhardee9) [inline app/helpers/service_calendar_helper.rb]: app/views/service_requests/show.xlsx.axlsx:277
app/views/service_requests/show.xlsx.axlsx:279

**Comment by SamWord** [comment]
I added the display_indirect_costs helper methods back for now, but I should check that Wenjun wants indirect totals to be displayed anywhere before properly removing them.

**Pull request merged**